aws/dms/cron-project-hierarchy.json

```diff
@@ -0,0 +1,48 @@
+{
+  "rules": [
+    {
+      "rule-type": "selection",
+      "rule-id": "1",
+      "rule-name": "1",
+      "object-locator": {
+        "schema-name": "pegasus",
+        "table-name": "storage_apps"
+      },
+      "rule-action": "include"
+    },
+    {
+      "rule-type": "selection",
+      "rule-id": "2",
+      "rule-name": "2",
+      "object-locator": {
+        "schema-name": "pegasus",
+        "table-name": "user_storage_ids"
+      },
+      "rule-action": "include"
+    },
+    {
+      "rule-type": "transformation",
+      "rule-id": "3",
+      "rule-name": "3",
+      "rule-target": "column",
+      "object-locator": { 
+        "schema-name": "pegasus",
+        "table-name": "storage_apps",
+        "column-name": "value"
+      },
+      "rule-action": "remove-column"
```

Comment:
How are you planning to determine whether a row represents a channel-backed level (as opposed to a standalone project)? The `hidden` field in the `value` column is where this bit lives today. The two ways I could see are (1) do a migration and add a new column, or (2) have redshift key off of whether `"hidden": true` or `"hidden": "true"` appear in the `value` field. Removing this column suggests the former, just wanting to confirm.